sdk/python/jobs/automl-standalone-jobs/automl-forecasting-recipes-univariate/automl-forecasting-recipe-univariate-run.ipynb

```diff
@@ -0,0 +1,1315 @@
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "#### Show Azure ML Workspace information"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "credential = DefaultAzureCredential()\n",
+    "ml_client = None\n",
+    "# import pdb;pdb.set_trace()\n",
+    "try:\n",
+    "    ml_client = MLClient.from_config(credential)\n",
+    "except Exception as ex:\n",
+    "    print(ex)\n",
+    "    # Enter details of your AML workspace\n",
+    "    subscription_id = \"<SUBSCRIPTION_ID>\"\n",
+    "    resource_group = \"<RESOURCE_GROUP>\"\n",
+    "    workspace = \"<AML_WORKSPACE_NAME>\"\n",
+    "    ml_client = MLClient(credential, subscription_id, resource_group, workspace)"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "workspace = ml_client.workspaces.get(name=ml_client.workspace_name)\n",
+    "\n",
+    "output = {}\n",
+    "output[\"Workspace\"] = ml_client.workspace_name\n",
+    "output[\"Subscription ID\"] = ml_client.connections._subscription_id\n",
+    "output[\"Resource Group\"] = workspace.resource_group\n",
+    "output[\"Location\"] = workspace.location\n",
+    "output"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## 2. Data\n",
+    "\n",
+    "We will load the data into DataFrame objects, split the data to train and test datasets, then create the Azure Machine Learning MLTable objects to prepare for the later training and inference steps."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### 2.1 Load the data file into DataFrame\n",
+    "\n",
+    "We will load the data on local machine in dataframe objects. Only for this experiment, the data is in data folder, uploaded manually. We will drop the Covid period."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "TARGET_COLNAME = \"S4248SM144SCEN\"\n",
+    "TIME_COLNAME = \"observation_date\"\n",
+    "COVID_PERIOD_START = (\n",
+    "    \"2020-03-01\"  # start of the covid period. To be excluded from evaluation.\n",
+    ")\n",
+    "\n",
+    "df = pd.read_csv(\"./data/S4248SM144SCEN.csv\", parse_dates=[TIME_COLNAME])\n",
+    "# df[TIME_COLNAME] = pd.to_datetime(df[TIME_COLNAME], format=\"%Y-%m-%d\")\n",
+    "df.sort_values(by=TIME_COLNAME, inplace=True)\n",
+    "\n",
+    "# remove the Covid period\n",
+    "df = df.query('{} <= \"{}\"'.format(TIME_COLNAME, COVID_PERIOD_START))"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "#### Set parameters"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "The first set of parameters is based on the analysis performed in the auto-ml-forecasting-univariate-recipe-experiment-settings notebook."
```

Comment:
Please highlight the notebook name. See the V1 version as an example.

Reply:
Done. added `auto-ml-forecasting-univariate-recipe-experiment-settings`.